src/stream/src/executor/managed_state/join/mod.rs

```diff
@@ -1,694 +0,0 @@
-// Copyright 2024 RisingWave Labs
```

Comment:
Just curious why these 3 files are deleted?

Reply:
I forgot to delete it last time...